src/modules/pm.c

```diff
@@ -122,8 +148,14 @@ static int on_session_change(UNUSED sd_bus_message *m, UNUSED void *userdata, UN
 
 static int parse_bus_reply(sd_bus_message *reply, const char *member, void *userdata) {
     int r = -EINVAL;
-    if (!strcmp(member, "Inhibit")) {
+    if (strcmp(member, "Inhibit") != 0) {
```

Comment:
Moreover, note that since `parse_bus_reply` callback is only called when acquiring the lock, i think we can drop the old branch of the if else statement.

Reply:
No it was not `==0`although, it should have been. But your larger point stands, I'll remove this

Reply:
```
if (!strcmp(member, "Inhibit")) {
```
means
```
if (strcmp(member, "Inhibit") == 0) {
```
:P 
Thanks btw!